docker-compose.yml

```diff
@@ -25,8 +25,9 @@ services:
       - 'sparkswapd:/data'
       # This is populated externally w/ an engine
       - 'shared:/shared'
-      # Persistent certs for broker
-      - './certs:/secure'
+      # Persistent certs/keys for broker
+      - ${SECURE_PATH}:/secure
```

Comment:
@martineehrlich just curious, did you get any more info on `""` vs `''` vs no quotes?